includes/media-backgrounds.php

```diff
@@ -66,7 +66,11 @@ function ucfwp_get_media_background_picture_srcs( $attachment_xs_id, $attachment
  * @return string
  **/
 if ( ! function_exists( 'ucfwp_get_media_background_picture' ) ) {
-	function ucfwp_get_media_background_picture( $srcs ) {
+	function ucfwp_get_media_background_picture( $srcs, $image_alt ) {
```

Comment:
It's possible to define a default value in the function definition.

```suggestion
	function ucfwp_get_media_background_picture( $srcs, $image_alt = 'Header background image' ) {
```

This removes the need for the extra empty/null check and default value setting.